whylabs_toolkit/helpers/schema.py

```diff
@@ -11,9 +11,33 @@
 BASE_ENDPOINT = "https://api.whylabsapp.com"
 
 
+@dataclass
+class ColumnsClassifiers:
+    inputs: List[str] = field(default=None)  # type: ignore
+    outputs: List[str] = field(default=None)  # type: ignore
+
+    def __post_init__(self) -> None:
+        if self.inputs is None:
+            self.inputs = []
+        if self.outputs is None:
+            self.outputs = []
```

Comment:
is this required? Can't you just make it default to []?

Reply:
not required, and I just found out that there's a special argument on `field` to address mutable default values for dataclasses :) https://docs.python.org/3/library/dataclasses.html#mutable-default-values
looks cleaner now